.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+jobs:
+   build:
+     docker:
+       - image: ubuntu:xenial
+     steps:
+       - checkout
+       - run:
+           name: Install system requirements
+           command: |
+             apt-get update
+             apt-get install -y software-properties-common python-software-properties
+       - run:
+           name: Install python3.6 and pip
+           command: |
+             add-apt-repository ppa:jonathonf/python-3.6
+             apt-get update
+             apt-get install -y python-pip python3.6
```

Comment:
I would now recommend to look into using the manylinux containers to run the test with.

The provided by dockcross should work well

Reply:
Thinking more about it ... we can skip this now, the project being tested is pure python

Reply:
Sounds good. I can do it in another PR later.